core/scss/utilities/mixins/layout/_sticky-footer.scss

```
@@ -0,0 +1,27 @@
+@charset 'UTF-8';
+
+//
+// @sticky-footer
+//
+// Slam the footer to the bottom of the window below all content no matter
+// the size of the browser or content.
+//
+// Apply this mixin to the body tag only. It will set the body tag height to
```

```suggestion
// Apply this mixin to the `<body>` tag only. It will set the body tag height to
```

Just a little pedantry. :)